torch/csrc/jit/python/init.cpp

```diff
@@ -1349,14 +1361,15 @@ void initJITBindings(PyObject* module) {
             IValue v = *self.default_value();
             return toPyObject(std::move(v));
           })
-      .def(
+      .def_property_readonly(
           "has_default_value",
           [](Argument& self) -> py::bool_ {
             return self.default_value().has_value();
           })
       .def_property_readonly("kwarg_only", [](Argument& self) -> bool {
-        return self.kwarg_only();
-      });
+        return self.kwarg_only();})
+      .def_property_readonly(
+          "is_mutable", [](Argument& self) -> bool { return self.alias_info() && self.alias_info()->isWrite(); });
```

Comment:
If we add a method here can we mirror it in the C++ API? Let's keep the APIs lined up as much as possible.

Reply:
sounds good